automation/src/test/scala/org/broadinstitute/dsde/test/api/SnapshotAPISpec.scala

```diff
@@ -215,6 +220,10 @@ class SnapshotAPISpec extends AnyFreeSpecLike with Matchers with BeforeAndAfterA
           val workspaceName = s"${UUID.randomUUID().toString}-delete-bq-dataset"
           Rawls.workspaces.create(projectName, workspaceName)
 
+          // we need the actual Google project for this workspace, not just its namespace
+          val googleProject = Rawls.workspaces.getWorkspaceDetails(projectName, workspaceName)(ownerAuthToken)
+            .parseJson.convertTo[WorkspaceResponse].workspace.googleProject
+
```

Comment:
Might be nice to convert this snippet to a function since we use it twice

Reply:
done.